operator/scripts/Dockerfile-custom-image

```diff
@@ -2,4 +2,4 @@ ARG IMAGE=keycloak
 ARG VERSION=latest
 FROM $IMAGE:$VERSION
 
-RUN /opt/keycloak/bin/kc.sh build --db=postgres --health-enabled=true
```

Comment:
Feature enabled in the custom/optimized image to make the test pass :crossed_fingers: